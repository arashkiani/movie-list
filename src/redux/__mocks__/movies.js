import { actionSortMovies } from '../filters'
import genresMocks from '../../mocks/genres'
import moviesMocks from '../../mocks/movies'
export const actionGetMovies = () => {
	return async dispatch => {
		const { page, total_pages, results } = await new Promise(resolve => {
			resolve(moviesMocks)
		})
		dispatch({
			type: 'getMovies',
			result: {
				list: results,
				pagination: { page, total_pages },
			},
		})
		dispatch(actionSortMovies())
	}
}
export const actionGetGenres = page => {
	return async dispatch =>
		dispatch({
			type: 'getGenres',
			result: await new Promise(resolve => {
				resolve(genresMocks)
			}),
		})
}

export const reducers = {
	getMovies:(state, {list, pagination}) => ({
		...state,
		list: list,
		pagination: pagination
	}),
	sortedList:(state, result) => {
		return ({
			...state,
			sortedMovies: result,
		})
	},
	getGenres:(state, result) => ({
		...state,
		genres: result.genres
	})
}
