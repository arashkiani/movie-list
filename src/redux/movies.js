import defaultState from './movies.default'
import zFetch from '../services/zFetch'
import {actionSortMovies} from './filters'
export const actionGetMovies = (page)=>{
	const movieFetch = new zFetch()
	//returns : results, total_pages, total_results, page
  return async (dispatch)=>{
		const {page, total_pages, results} = await movieFetch.getList(1)
		dispatch({
			type: 'getMovies',
			result: {
				list: results, 
				pagination: {page, total_pages}
			},
		})
		dispatch(actionSortMovies())
	}
}
export const actionGetGenres = ()=>{
	const movieFetch = new zFetch()
  return async dispatch=>dispatch({
		type: 'getGenres',
		result: await movieFetch.getGenres(),
	})
}

const reducers = {
	getMovies:(state, {list, pagination}) => ({
		...state,
		list: list,
		pagination: pagination
	}),
	sortedList:(state, result) => {
		return ({
			...state,
			sortedMovies: result,
		})
	},
	getGenres:(state, result) => ({
		...state,
		genres: result.genres
	})
}

export default (state = defaultState, action) => {
	const { type, result } = action
	return !reducers[type]?state:reducers[type](state, result)
}