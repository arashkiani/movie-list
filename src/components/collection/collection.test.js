/*global shallow, mount, sinon */
import {Collection} from '.'
import genres from '../../mocks/genres'
import movies from '../../mocks/movies'

const mapToProps = {
	sortedMovies: movies.results,
	getMovies: jest.fn(() => ({})),
}

let mountWrapper;
beforeEach(() => {
	mountWrapper = mount(<Collection store={store} {...mapToProps}/>)
})
describe('<Collection />', () => {
	it('Match Snapshot', () => {
		expect(mountWrapper).toMatchSnapshot()
	})
	it('Number of movies should be ' + movies.results.length, ()=>{
		expect(mountWrapper.find('movie').length).toBe(movies.results.length)
	})
})