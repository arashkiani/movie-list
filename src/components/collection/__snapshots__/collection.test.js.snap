// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Collection /> Match Snapshot 1`] = `
<ForwardRef>
  <div
    className="sc-EHOje hptrzz"
  >
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The Nun Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/sFC1ElvoKGdHJIWRpNB3xWJ9lJA.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The Nun
                </h2>
                6.2
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                When a young nun at a cloistered abbey in Romania takes her own life, a priest with a haunted past and a novitiate on the threshold of her final vows are sent by the Vatican to investigate. Together they uncover the order’s unholy secret. Risking not only their lives but their faith and their very s
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The Meg Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/xqECHNvzbDL5I3iiOVUkVPJMSbc.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The Meg
                </h2>
                6.2
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                A deep sea submersible pilot revisits his past fears in the Mariana Trench, and accidentally unleashes the seventy foot ancestor of the Great White Shark believed to be extinct.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The Seven Deadly Sins: Prisoners of the Sky Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/r6pPUVUKU5eIpYj4oEzidk5ZibB.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The Seven Deadly Sins: Prisoners of the Sky
                </h2>
                5.2
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                The Seven Deadly Sins travel to a remote land in search of the phantom ingredient "sky fish." Meliodas and Hawk end up at a "Sky Palace" that exists above the clouds, where all the residents have wings. Meliodas is mistaken for a boy who committed a crime and is thrown in prison. Meanwhile, the resi
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The Predator Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/wMq9kQXTeQCHUZOG4fAe5cAxyUA.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The Predator
                </h2>
                5.1
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                From the outer reaches of space to the small-town streets of suburbia, the hunt comes home. Now, the universe’s most lethal hunters are stronger, smarter and deadlier than ever before, having genetically upgraded themselves with DNA from other species. When a young boy accidentally triggers their re
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Mission: Impossible - Fallout Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/AkJQpZp9WoNdj7pLYSj1L0RcMMN.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Mission: Impossible - Fallout
                </h2>
                7.4
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                When an IMF mission ends badly, the world is faced with dire consequences. As Ethan Hunt takes it upon himself to fulfil his original briefing, the CIA begin to question his loyalty and his motives. The IMF team find themselves in a race against time, hunted by assassins while trying to prevent a gl
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The First Purge Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/2slvblTroiT1lY9bYLK7Amigo1k.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The First Purge
                </h2>
                6
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                To push the crime rate below one percent for the rest of the year, the New Founding Fathers of America test a sociological theory that vents aggression for one night in one isolated community. But when the violence of oppressors meets the rage of the others, the contagion will explode from the trial
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Ant-Man and the Wasp Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/rv1AWImgx386ULjcf62VYaW8zSt.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Ant-Man and the Wasp
                </h2>
                7
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                As Scott Lang awaits expiration of his term of house detention, Hope van Dyne and Dr. Hank Pym involve him in a scheme to rescue Mrs. van Dyne from the micro-universe into which she has fallen, while two groups of schemers converge on them with intentions of stealing Dr. Pym's inventions.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="A Simple Favor Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/aIYsjQM0piKc0ibPBCTWv3sqshj.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  A Simple Favor
                </h2>
                5.7
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Mommy vlogger Stephanie joins her best friend Emily's husband, Sean, to investigate her sudden disappearance from their small town.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Alpha Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/afdZAIcAQscziqVtsEoh2PwsYTW.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Alpha
                </h2>
                5.3
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                After a hunting expedition goes awry, a young caveman struggles against the elements to find his way home.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Hereditary Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/dscvG9AWLOzxedHcXN7QguJRhsP.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Hereditary
                </h2>
                6.9
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                When Ellen, the matriarch of the Graham family, passes away, her daughter's family begins to unravel cryptic and increasingly terrifying secrets about their ancestry.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Incredibles 2 Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/x1txcDXkcM65gl7w20PwYSxAYah.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Incredibles 2
                </h2>
                7.7
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Elastigirl springs into action to save the day, while Mr. Incredible faces his greatest challenge yet – taking care of the problems of his three children.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Upgrade Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/8fDtXi6gVw8WUMWGT9XFz7YwkuE.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Upgrade
                </h2>
                7.3
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                A brutal mugging leaves Grey Trace paralyzed in the hospital and his beloved wife dead. A billionaire inventor soon offers Trace a cure — an artificial intelligence implant called STEM that will enhance his body. Now able to walk, Grey finds that he also has superhuman strength and agility — skills 
                ...
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Adrift Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/5gLDeADaETvwQlQow5szlyuhLbj.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Adrift
                </h2>
                6.4
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                A true story of survival, as a young couple's chance encounter leads them first to love, and then on the adventure of a lifetime as they face one of the most catastrophic hurricanes in recorded history.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="The Equalizer 2 Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/cQvc9N6JiMVKqol3wcYrGshsIdZ.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  The Equalizer 2
                </h2>
                6.3
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Robert McCall returns to deliver his special brand of vigilante justice -- but how far will he go when it's someone he loves?
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Mandy Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/m0yf7J7HsKeK6E81SMRcX8vx6mH.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Mandy
                </h2>
                6.3
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Pacific Northwest. 1983 AD. Outsiders Red Miller and Mandy Bloom lead a loving and peaceful existence. When their pine-scented haven is savagely destroyed by a cult led by the sadistic Jeremiah Sand, Red is catapulted into a phantasmagoric journey filled with bloody vengeance and laced with fire.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Peppermint Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/jrzxS0vcbzIIay1sdYm0rgI2QfJ.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Peppermint
                </h2>
                7.1
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                A grieving mother transforms herself into a vigilante following the murders of her husband and daughter, eluding the authorities to deliver her own personal brand of justice.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Crazy Rich Asians Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/gnTqi4nhIi1eesT5uYMmhEPGNih.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Crazy Rich Asians
                </h2>
                6.9
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                An American-born Chinese economics professor accompanies her boyfriend to Singapore for his best friend's wedding, only to get thrust into the lives of Asia's rich and famous.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="BlacKkKlansman Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/55W6mUVv4CXMMQHHhV2zXtLSpXQ.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  BlacKkKlansman
                </h2>
                7.5
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Ron Stallworth, an African-American police officer from Colorado, successfully manages to infiltrate the local Ku Klux Klan and become the head of the local chapter.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="Sicario: Day of the Soldado Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/OdD7A0LgLXPjqzmfcEl9sTYVPo.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  Sicario: Day of the Soldado
                </h2>
                6.8
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                Agent Matt Graver teams up with operative Alejandro Gillick to prevent Mexican drug cartels from smuggling terrorists across the United States border.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
    <ForwardRef>
      <div
        className="sc-bdVaJa YQLtG"
      >
        <ForwardRef>
          <div
            className="sc-bwzfXH ipjmxz"
          >
            <img
              alt="White Boy Rick Poster Image"
              src="https://image.tmdb.org/t/p/w185_and_h278_bestv2/376lzYd3vE04qBA5Js93UAKrnvj.jpg"
            />
          </div>
        </ForwardRef>
        <ForwardRef>
          <div
            className="sc-htpNat hPWpwX"
          >
            <ForwardRef>
              <div
                className="sc-bxivhb bMtJor"
              >
                <h2>
                  White Boy Rick
                </h2>
                6
              </div>
            </ForwardRef>
            <ForwardRef>
              <div
                className="sc-ifAKCX eikNCR"
              >
                The story of teenager, Richard Wershe Jr., who became an undercover informant for the police during the 1980s and was ultimately arrested for drug-trafficking and sentenced to life in prison.
              </div>
            </ForwardRef>
          </div>
        </ForwardRef>
      </div>
    </ForwardRef>
  </div>
</ForwardRef>
`;
